source/simulator/parameters.cc

```diff
@@ -309,6 +310,11 @@ namespace aspect
     // to indicate a boundary
     prm.enter_subsection ("Model settings");
     {
+      prm.declare_entry ("Include melt transport", "false",
+                         Patterns::Bool (),
+                         "Whether to include the transport of melt into the model or not. If this "
+                         "is set to true, an additional compositional field for the porosity and "
+                         "an additional pressure (the compaction pressure) will be added.");
```

Comment:
This description is a bit contradictive to the assert in the subsection "Compositional fields" where you state that the user has to create at least one compositional field. Do you require the user to add the compositional field herself (by increasing the "Number of fields"), or will it be done automatically? I do not care either way, as long as it is described here.